docs/UsersGuide/source/ConfigWorkflow.rst

```diff
@@ -1222,6 +1225,48 @@ Set parameters for customizing the :term:`UPP`.
    
    Note that this variable is first changed to lower case before being used to construct the file names.
 
+RUN_PRDGEN Configuration Parameters
+=====================================
+
+Non-default parameters for the ``run_prdgen`` task are set in the ``task_run_prdgen:`` section of the ``config.yaml`` file.
+
+Basic Task Parameters
+---------------------------------
+For each workflow task, certain parameter values must be passed to the job scheduler (e.g., Slurm), which submits a job for the task.
+
+``TN_RUN_PRDGEN``: (Default: "run_prdgen")   Set the name of this Rocoto workflow task. Users typically do not need to change this value.
```

Comment:
@BenjaminBlake-NOAA I think you might need a new line before the definition on this one.

Reply:
Thanks for catching that, I'll get it fixed.